core/templates/dev/head/app.js

```diff
@@ -93,6 +93,10 @@ oppia.config(['$provide', function($provide) {
   }]);
 }]);
 
+oppia.factory('checkIfMobileDevice', [function() {
+  return typeof window.orientation !== 'undefined';
```

Comment:
I think it's better practice to use Angular's $window instead (and inject that as a dependency).

Also can you add a comment with a reference link for this approach? I'm guessing it's [this one](http://stackoverflow.com/a/14301832)?